app/javascript/components/cloud-volume-form/index.jsx

```diff
@@ -0,0 +1,75 @@
+import React, { useState, useMemo, useEffect } from 'react';
+import PropTypes from 'prop-types';
+
+import MiqFormRenderer from '@@ddf';
+import createSchema from './cloud-volume-form.schema';
+import miqRedirectBack from '../../helpers/miq-redirect-back';
+
+const CloudVolumeForm = ({ recordId }) => {
+  const [{ fields, initialValues, isLoading }, setState] = useState({ fields: [], isLoading: !!recordId });
+  const submitLabel = !!recordId ? __('Save') : __('Add');
+  const promise = useMemo(() =>
+    API.get('/api/providers?expand=resources&attributes=id,name,supports_block_storage&filter[]=supports_block_storage=true'), []);
+
+  const loadSchema = (value, appendState = {}) => {
+    console.log(value);
```

Comment:
The `value` will be no longer needed, the function can have just a single argument.